_includes/com-config/com-config-dataframe-summary.md

```diff
@@ -0,0 +1,3 @@
+The Dataframe data format is an alternative to the native bitmap format used in FeatureBase by default.
```

Comment:
Let's get rid of this includes. Dataframe is something for a single customer that is likely going to go away